service/http/http.go

```diff
@@ -21,34 +21,26 @@ func New() *httpService {
 	return &httpService{}
 }
 
+// GetJobs function
 func (h *httpService) GetJobs() ([]model.ExtJob, error) {
 	client := &http.Client{}
-
 	req, err := http.NewRequest("GET", "http://api.dataatwork.org/v1/jobs", nil)
```

Comment:
```suggestion
	req, err := http.NewRequest(http.MethodGet, h.externalURL, nil)
```
Again, no magic naming, make use of already defined constants.
I'd suggest not hardcode these URLs, it's better if you consider it as a dependency, which should be a part of your httpService struct. This way, if in the future this URL changes, you just change it from the main.

These values should be either taken from a config file, or from an environment variable.